### ISSUE TITLE
Accordion content section should have accessibility roles

### ISSUE THREAD

**Issue by caroqliu** [ampproject/amphtml#30727]
[According to this example,](https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html) the content section should have `aria-labelledby` and `role=region`.
